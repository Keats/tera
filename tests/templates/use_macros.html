{% import "macros.html" as macros %}
{{ macros::input(label="Name", type="text") }}
{{ macros::input(label="Age", type="number") }}
{{ macros::input(label=username, type="text") }}

{{ macros::factorial(n=7) }}

{{ macros::include(greeting="Bonjour") }}
