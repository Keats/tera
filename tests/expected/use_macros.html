
<label>
    Name
    <input type="text" />
</label>
<label>
    Age
    <input type="number" />
</label>
<label>
    bob
    <input type="text" />
</label>

7 - 6 - 5 - 4 - 3 - 2 - 11234567

Bonjour humans
